mysqlbkup.sh

```diff
@@ -84,7 +84,10 @@ do
 done
 
 # the date is used for backup file names
-date=$(date +%F)
+date =$(date +%F)
```

Comment:
Is the space here intentional? I think this line should remain as it was `date=$(date +%F)`

Reply:
@quickshiftin it's type. I have fixed.